io/net/sriov_device_test.py

```diff
@@ -70,14 +70,18 @@ def setUp(self):
                                              '*', default=None).split(' ')
         self.sriov_port = self.params.get('sriov_port', '*',
                                           default=None).split(' ')
-        self.ipaddr = self.params.get('ipaddr', '*', default="").split(' ')
+        self.max_sriov_port = self.params.get('max_sriov_ports')
+        self.ipaddr = self.params.get('ipaddr', '*', default='').split(' ')
```

Comment:
remove path parameter '*' from all params

Reply:
@abdhaleegit removed it and pushed the latest code.